releases/v21.1.0-rc.2.md

```diff
@@ -0,0 +1,50 @@
+---
+title: What&#39;s New in v21.1.0-rc.2
+toc: true
+summary: Additions and changes in CockroachDB version v21.1.0-rc.2 since version v21.1.0-rc.1
+---
+
+## May 05, 2021
+
+Get future release notes emailed to you:
+
+{% include marketo.html %}
+
+### Downloads
+
+<div id="os-tabs" class="filters clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v21.1.0-rc.2.linux-amd64.tgz"><button id="linux" class="filter-button" data-scope="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v21.1.0-rc.2.darwin-10.9-amd64.tgz"><button id="mac" class="filter-button" data-scope="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v21.1.0-rc.2.windows-6.2-amd64.zip"><button id="windows" class="filter-button" data-scope="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v21.1.0-rc.2.src.tgz"><button id="source" class="filter-button" data-scope="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+<section class="filter-content" data-scope="windows">
+{% include windows_warning.md %}
+</section>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v21.1.0-rc.2
+~~~
+
+### Enterprise edition changes
+
+- [Changefeeds](../v21.1/change-data-capture.html) more reliably fail when [`IMPORT INTO`](../v21.1/import-into.html) is run against a targeted table. [#64372][#64372]
```

Comment:
```suggestion
- [Changefeeds](../v21.1/change-data-capture.html) now reliably fail when [`IMPORT INTO`](../v21.1/import-into.html) is run against a targeted table, as change data capture is [not supported](../v21.1/known-limitations.html#change-data-capture) for this action. [#64372][#64372]
```

I know the dev wrote "more reliably fail" but in that doc I've linked and [this comment](https://github.com/cockroachdb/cockroach/issues/43784#issuecomment-828816650) it seems that we expect this to always fail, as we've intended.